src/utils/i18nTranslations.ts

```diff
@@ -34,28 +34,40 @@ export const TX_TYPE_TRANSLATIONS: TranslationMap<TxType> = {
   PayingForTx: () => tg('transaction.type.payingForTx'),
   SpendTx: () => tg('transaction.type.sentTx'),
 };
+export function getTxTypeLabel(txType: TxType): string {
+  const translateFunc = TX_TYPE_TRANSLATIONS[txType];
+  return translateFunc ? translateFunc() : '';
+}
 
 /**
  * Replacements for the `txTypeTranslations` displayed on the transaction lists
  * ITx.type
  */
-export const TX_TYPE_LIST_TRANSLATIONS: TranslationMap<TxType> = {
+const TX_TYPE_LIST_TRANSLATIONS: TranslationMap<TxType> = {
   NamePreclaimTx: () => tg('transaction.listType.namePreclaimTx'),
   NameClaimTx: () => tg('transaction.listType.nameClaimTx'),
   NameUpdateTx: () => tg('transaction.listType.nameUpdateTx'),
   NameTransferTx: () => tg('transaction.listType.nameTransferTx'),
   NameRevokeTx: () => tg('transaction.listType.nameRevokeTx'),
   SpendTx: () => tg('transaction.listType.sentTx'),
 };
+export function getTxTypeListLabel(txType: TxType): string {
+  const translateFunc = TX_TYPE_LIST_TRANSLATIONS[txType];
+  return translateFunc ? translateFunc() : '';
+}
 
 /**
  * ITx.function translated into human readable label
  */
-export const TX_FUNCTION_TRANSLATIONS: TranslationMap<TxFunctionRaw | TxFunctionMultisig> = {
+const TX_FUNCTION_TRANSLATIONS: TranslationMap<TxFunctionRaw | TxFunctionMultisig> = {
   propose: () => tg('transaction.function.propose'),
   revoke: () => tg('transaction.function.revoke'),
   refuse: () => tg('transaction.function.refuse'),
   confirm: () => tg('transaction.function.confirm'),
   tip_token: () => tg('transaction.function.tip_token'),
   retip_token: () => tg('transaction.function.retip_token'),
 };
+export function getTxFunctionLabel(txFunction: TxFunctionRaw | TxFunctionMultisig): string {
+  const translateFunc = TX_FUNCTION_TRANSLATIONS[txFunction];
+  return translateFunc ? translateFunc() : '';
```

Comment:
```suggestion
  return TX_FUNCTION_TRANSLATIONS[txFunction] ? translateFunc() : '';
```

Reply:
This change makes no sense as you first suggest to remove the `translateFunc` constant and than you use it.